### PR TITLE
Increased code coverage (merge super-coverage into nightly)

### PR DESCRIPTION
**Pull request by quantum9Innovation**
## Code Coverage Info
 - Achieved 98.27% code coverage
    - Excluding the `Idiom` class (for debugging purposes only)
    - Excludes `verbose` logging on `plots.py` "
 - Added multiple new testing files
 - Some workflows changed

**Before Merging:**
 - Change workflows to check for pushes *only* to the `nightly` branch
 - Change README.md badge
 - Scan for any other mentions to `super-coverage` in workflow and repo files

## Review Process
This PR will be merged after code review from @Quantalabs
All checks (other than codefactor.io and LGTM code quality analyses) must pass in order for this PR to be merged

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 [![Downloads](https://pepy.tech/badge/epispot-nightly)](https://pepy.tech/project/epispot-nightly)
 ![build-status](https://github.com/epispot/epispot/workflows/build/badge.svg?branch=nightly)
 ![open-issues](https://img.shields.io/github/issues-search/epispot/epispot?color=red&label=Open%20Issues&query=is%3Aopen%20label%3Anightly)
-[![codecov](https://codecov.io/gh/epispot/epispot/branch/nightly/graph/badge.svg?token=WGIM127RFY)](https://codecov.io/gh/epispot/epispot)
+[![codecov](https://codecov.io/gh/epispot/epispot/branch/super-coverage/graph/badge.svg?token=WGIM127RFY)](https://codecov.io/gh/epispot/epispot)
```
Comment (Quantalabs) [inline README.md:7]: NOTE: Do not merge this change when merging into nightly.

File: tests/CI/test_cov.py [hunk 2]
```diff
@@ -1,2 +1,3 @@
-print('This code coverage test has successfully been reached. YAY!')
+print('This code coverage test has successfully been reached.')
```
Comment (Quantalabs) [inline tests/CI/test_cov.py:1]: What is the point of this file?

**Review by Quantalabs — CHANGES_REQUESTED** [review]
I've fixed the Code Factor error. View my comments on both the `README.md` file and `test_cov.py`.

File: tests/CI/test_cov.py (hunk 2 above)
Comment (quantum9Innovation) [reply tests/CI/test_cov.py:1]: Triggers code coverage reports

File: README.md (hunk 1 above)
Comment (quantum9Innovation) [reply README.md:7]: How do we do that?

Comment (Quantalabs) [reply README.md]: This has been fixed already. You can merge the PR now. Squashing is suggested.

**Pull request merged**